cmd/node/node.go

```diff
@@ -1164,3 +1170,35 @@ func decodeLoggers(cfg config.LoggerConfig) (map[string]string, error) {
 	}
 	return rst, nil
 }
+
+func (app *App) checkAndStoreGenesisConfig() error {
+	genesisConfigPath := filepath.Join(app.Config.Genesis.Path, config.DefaultGenesisConfigFileName)
```

Comment:
this should use configured data dir, not some new parameter

Reply:
fixed